### ISSUE TITLE
Old `sigplan` bib formatting code emitted even when `nosigplan` specified

### ISSUE THREAD

**Issue by jsarracino** [wilbowma/mttex#44]
Unfortunately it appears that mttex breaks the SIGPLAN bibliography style. I've attached a minimal example [sigplan-empty.zip](https://github.com/wilbowma/mttex/files/2498238/sigplan-empty.zip) that shows the issue -- to reproduce, run `pdflatex empty && bibtex empty && pdflatex empty`.

I see the error message:

```console
(./empty.bbl
! Argument of \markboth has an extra }.
<inserted text> 
                \par 
l.5 \begin{thebibliography}{1}
```
EDIT: updated example to include original mttex.sty (and not a symlink).

**Comment by wilbowma** [comment]
Looks like this happens because the acmart `sigplan` document class option gets picked up by mttex.
Think I should just remove some of the conditional formatting crap for sigplan, since it's mostly built-in to acmart.

**Comment by wilbowma** [comment]
Closed by a310b5b653466b45f978dc7eacd80487e473f54e
